sdk/core/azure-core/src/main/java/com/azure/core/implementation/util/FluxUtil.java

```diff
@@ -61,7 +61,8 @@ private static void accept(ByteArrayOutputStream byteOutputStream, ByteBuffer by
         try {
             byteOutputStream.write(byteBufferToArray(byteBuffer));
         } catch (IOException e) {
-            throw new RuntimeException(e);
+            throw new RuntimeException("Error occurs when write ByteBuffer to ByteArrayOutputStream. "
+                + "Error details: " + e.getMessage(), e);
```

Comment:
Should not have to print these error details. It'll be duplicated in the inner throwable. Same with one below,.

Reply:
If not logged out the error message, then it lost the IOException details.

Reply:
It's a problem with our ClientLogger, not this logic.